cmd/caa-log-checker/main.go

```diff
@@ -174,6 +174,8 @@ func loadMap(paths []string) (map[string][]time.Time, error) {
 }
 
 func main() {
+	logStdoutLevel := flag.Int("stdout-level", 7, "Minimum severity of messages to send to stdout")
```

Comment:
Do we want to default to stdout-only? I thought when we run this in prod we want its output going to syslog?

Reply:
Yes, but then I can pass the proper flag. In general I think it makes sense to set up the defaults geared towards interactive usage. (Otherwise you need to keep passing flags for testing, whereas the production deploy is going to use a unit file in all cases anyway.)

Reply:
You can set a nonzero logStdoutLevel and also a nonzero logSyslogLevel, which is what most Boulder components are configured to do in test/config.

Reply:
Sure. I'll still be passing the flags anyway though (to disable the stdout logging and prevents duplicates.)

(Personally, I feel like with a tool like this that's read-only, it seems desirable to not enable syslog unless for "official" runs, because otherwise you'd just end up polluting the logs with confusing messages about a one-off/test run that doesn't actually affect the rest of the system in any way. Calculus probably changes depending on the scope of the tool I suppose.)

Anyway, this is a silly thing to nitpick about so I'll keep the settings uniform with the rest of boulder.